src/Unidevel.OpenWeather/Unidevel.OpenWeather.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <Version>0.9.5</Version>
```

Comment:
Upgrade version number everywhere to 0.10.0. We will release nuget package at the end of our work.